### ISSUE TITLE
Hatch preview in "Choose Hatch Attributes" window is broken

### ISSUE THREAD

**Issue by app4soft** [LibreCAD/LibreCAD#1701]
### Expected behavior

Preview should be as expected: square in a half of window width.

### Observed behavior

Actual preview width stretched to thin vertical rectangle, see screenshot:

![Screenshot_2023-10-12_19-37-29](https://github.com/LibreCAD/LibreCAD/assets/22447924/06850fc7-4eee-42f9-a83a-6766a16db03b)

### Steps to reproduce or sample file

1. Launch latest LibreCAD `2.2.1-alpha` AppImage;
2. Create rectangle (or any shape) and select it;
3. Launch "Hatch" tool.

### Operating System and LibreCAD version info

```
Version: 2.2.1_alpha-298-g5e701b71
Compiler: GNU GCC 11.4.0
Compiled on: Oct  7 2023
Qt Version: 5.15.3
Boost Version: 1.74.0
System: Debian GNU/Linux 12 (bookworm)
```

**Comment by gerardbm** [comment]
After the commit `f77eedc` I downloaded the last continuus build to check the improvements. Sometimes, after a double click into the hatch area, the dialog window is not opened; instead of it, LibreCAD crashed with the msg in the terminal `[1]    9746 segmentation fault  ./LibreCAD-2.2.1_alpha-299-gf77eedc5-x86_64.AppImage`.

Btw, I'm not sure if I'm missing something, but doing a square of 1x1 meter, I had to input the hatch with a scale of 0.01 to view a single line. If one can't see a line of a hatch, then it's unselectable (probably it was not created).
